NLP/Quora/RNN.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "metadata": {
```

Comment:
@Xerefic  No need to have 2 separate notebooks, both are exactly the same except the model line, keep only 1 notebbok